.travis.yml

```diff
@@ -1,37 +1,44 @@
-sudo: false
+os: linux
 language: scala
 stages:
   - name: test
   - name: release
     if: (branch = master AND type = push) OR (tag IS present)
 jobs:
   include:
-  - script: "./mill __.jvm[2.12.7].test"
+  - script: "./mill __.jvm[2.12.10].test"
     jdk: oraclejdk9
 
-  - script: "./mill __.jvm[2.13.0].test"
+  - script: "./mill __.jvm[2.13.1].test"
     jdk: openjdk10
 
-  - script: "./mill __.js[2.12.7].test"
+  - script: "./mill __.js[2.12.10,_].test"
     jdk: openjdk10
 
-  - script: "./mill __.js[2.13.0].test"
+  - script: "./mill __.js[2.13.1,_].test"
```

Comment:
I think there has been a breaking change in mill here. Because you can't refer directly without expressing a wildcard for the other parameters..